client/src/ui/molecules/OrcidDetails.tsx

```diff
@@ -17,6 +18,7 @@ const OrcidDetails = ({ getOrcidDetails }: Props, authorDetails: AuthorDetails):
     const [authorLastName, setAuthorLastName] = useState(authorDetails.authorLastName || '');
     const [authorEmail, setAuthorEmail] = useState(authorDetails.authorEmail || '');
     const [institution, setInstitution] = useState(authorDetails.institution || '');
+    const { t } = useTranslation();
```

Comment:
Should we be loading translations internally or passing down strings for the values to this component?

Reply:
my view is that because this is a component group specifically for Orcid (currently) these strings will be the same no matter what but can be overridden in the config files so makes sense to set them internally to reduce complexity.